docs/technical/tests/frontend/api.rst

```diff
@@ -0,0 +1,6 @@
+API testing
+===============
+
+Currently mongodb needs to be running for FE tests. To remove this barrier and potential bottleneck we're moving away from using fetch, to use Axios. It provides us with convinient methods and a single place for API call.  Axios Mock Adapter offers the possibility to mock all API requests and eventually remove mongodb as a dependency for FE related tests.  
```

Comment:
- `FE` -> spell out to `front end`
- for `fetch` place it in code quotes
- Link to the Axios package so documentation for it can be easily found
- `convinient` -> `convenient`